Customer-research/Customer-insights-file.md

```diff
@@ -14,6 +14,7 @@ Please use Mark Down to format your quote. Here's [a guide](https://guides.githu
 -Student Z
 
 ---
+## This is the heading 
 
-Zara text
+> "driving business decisions and working with stakeholders. I like using Tamara Adlin's job hunting tool that has two weeks and two books away from being able to do something. This helps me identify jobs I can apply to because I believe I could pick up the skills I am missing."
```

Comment:
this is super interesting - I wonder if we could share this tool with the recruiting team